### ISSUE TITLE
#{:question} Documentation unsync with the latest release?

### ISSUE THREAD

**Issue by niwinz** [xtdb/xtdb#333]
Hi, I'm trying to play a little bit with crux but i found that the release `juxt/crux-core {:mvn/version "19.07-1.3.0-alpha"}` (the `juxt/crux-jdbc` is also included in my deps) does not exposes the functions used in the documentation.

I'm concretely trying to use jdbc related functions ( https://juxt.pro/crux/docs/configuration.html#_jdbc_nodes ) concretelly the `start-jdbc-node` is not found...

I'm missing something?

My deps.edn:

```clojure
{:paths ["src"]
 :deps {juxt/crux-core {:mvn/version "19.07-1.3.0-alpha"}
        juxt/crux-jdbc {:mvn/version "19.07-1.3.0-alpha"}
        org.xerial/sqlite-jdbc {:mvn/version "3.28.0"}}
 :aliases
 {:dev
  {:extra-deps {org.clojure/clojure {:mvn/version "1.10.1"}
                com.bhauman/rebel-readline {:mvn/version "0.1.4"}
                org.clojure/tools.namespace {:mvn/version "0.3.1"}}
   :extra-paths ["test"]}}}
```

**Comment by refset** [comment]
Hi @niwinz I just tested usage using the new `1.4.0` release.

In `crux-dev=> lein repl` I can run the following successfully:
```
(ns crux-jdbc-example
  (:require [crux.api :as crux])
  (:import (crux.api Crux ICruxAPI)))

(def opts {:dbtype "sqlite"
           :port 5432
           :dbname "sqlite-db-file"
           :user "sqlite"
           :kv-backend "crux.kv.memdb.MemKv"
           :db-dir "kv-store"})

(def node (crux/start-jdbc-node opts))
```

Would you mind trying again with the new version? It probably was the case that the docs were out of sync. We still need to figure out how to version the docs effectively (sorry!).

**Comment by niwinz** [comment]
yeah, I tested with the version from git/master and it works correctly. I just found 1.3.0 in the docs and then the usage example of `start-jdbc-node` that is not available in 1.3.0.

Now seems like docs are referencing 1.4.0 so, the issue with docs is solved. Thanks! :heart:
